internal/providers/terraform/google/service_networking_connection_test.go

```diff
@@ -0,0 +1,16 @@
+package google_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+)
+
+func TestServiceNetworkingConnection(t *testing.T) {
```

Comment:
FYI resourcegen command generates tests having `GoldenFile` postfix. I wonder if we should do that or actually drop it. What do you think? It makes the test explicit, but may be redundant, as it's still a regular test.
```suggestion
func TestServiceNetworkingConnectionGoldenFile(t *testing.T) {
```

Reply:
Oops, I'll stick with the convention for now.  I guess the suffix was more important when we were stilling converting everything to golden file, so it should be ok to remove now if we want.

Reply:
Sounds good, I'll change the resourcegen then. 👍